src/components/Navbar.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { HomeIcon } from '@heroicons/react/24/solid';
+import { User } from './User';
+
+const Navbar = () => (
+  <nav className="App-navbar">
+    <ul>
+      <li>
+        <NavLink to="/" className={({ isActive }) => `App-navbar__item ${isActive ? 'App-navbar__item--active' : ''}`}>
```

Comment:
This is long enough to wrap in my browser; mind if we wrap it?

```suggestion
        <NavLink
          to="/"
          className={({ isActive }) => (
            `App-navbar__item ${isActive ? 'App-navbar__item--active' : ''}`
          )}
        >
```

Alternatively, [a `<NavLink>` has a default `active` class](https://reactrouter.com/en/main/components/nav-link#default-active-class); is writing a rule for `.App-navbar__item a.active` a terrible idea?

Reply:
Oh, it defaults to that? Yes, I'm on board with that for sure.